exercises/practice/triangle/triangle.f90

```diff
@@ -0,0 +1,53 @@
+
+module triangle
+  implicit none
+
+  interface equilateral
+    module procedure equilateral_real
+    module procedure equilateral_int
+  end interface
+
+  interface scalene
+    module procedure scalene_real
+    module procedure scalene_int
+  end interface
+
+  interface isosceles
+    module procedure isosceles_real
+    module procedure isosceles_int
+  end interface
+
+ contains
+
+  logical function equilateral_real(traingle_edges)
```

Comment:
Thanks for adding these stubs!

"triangle" is misspelled here. (Also in several places in the example, but I care more about the student-facing material)

Also, thanks for letting me know the variable can't be named "triangle" - I didn't know in Fortran that local variables can't shadow a module name. Makes sense

What do you think about just using the word `edges` or `sides` for the function parameter? I think the name `triangle_edges` is repeating information that should be clear from context

Reply:
duh, thanks. Edit was too quick, sorry about that . "edges" is a much better name.

Reply:
Not a "duh" moment imo - lots of little changes can make concerns like this less obvious (esp around naming)